nrm_django/grants/models.py

```diff
@@ -361,6 +366,7 @@ class Grant(models.Model):
     )
     reroute_from = models.CharField(max_length=10, blank=True, null=True)
     reroute_date = models.DateField(blank=True, null=True)
+    # TODO: Is this typo in the source data?
```

Comment:
Yes, the field name in the source database is this. We could improve on it by calling the field `certification_date` and using `db_column="certificaion_date"`.

Reply:
Thanks. I didn't intend for this todo to make it in, but since we're here I may update that field name.